DOCS/contribute.md

```diff
@@ -224,15 +224,15 @@ New code must follow the following conventions:
 General coding
 --------------
 
-- Use C99. Also freely make use of C99 features if it's appropriate, such as
-  stdbool.h. (Except VLA and complex number types.)
+- Use C11. Also freely make use of C11 features if it's appropriate, but do not
+  use VLA and complex number types.
 - Don't use non-standard language (such as GNU C-only features). In some cases
   they may be warranted, if they are optional (such as attributes enabling
   printf-like format string checks). "#pragma once" is allowed as an exception.
-  But in general, standard C99 must be used.
+  But in general, standard C11 must be used.
 - The same applies to libc functions. We have to be Windows-compatible too. Use
-  functions guaranteed by C99 or POSIX only, unless your use is guarded by a
-  configure check. There is some restricted use of C11 (ask on IRC for details).
+  functions guaranteed by C11 or POSIX only, unless your use is guarded by a
+  configure check.
```

Comment:
suggestion:
```diff
+Be mindful of MinGW-specifics as C11 support is not always given.
```

an example for this is `timespec_get` which is only available with UCRT
maybe not so important as thankfully our CI will catch such uses.

Reply:
Went with `guaranteed` instead.